### DISCUSSION TITLE
Vendo mi coche mini

### DISCUSSION THREAD

**Discussion by izancabero** [community/community#144999] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

<!DOCTYPE html>
<html lang="es">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Decepción</title>
  <style>
    body {
      font-family: Arial, sans-serif;
      text-align: center;
      background-color: #f2f2f2;
      margin: 0;
      padding: 0;
    }
    h1 {
      color: #444;
    }
    .gallery {
      display: flex;
      flex-wrap: wrap;
      justify-content: center;
      gap: 10px;
      padding: 20px;
    }
    .gallery img {
      width: 200px;
      height: 200px;
      object-fit: cover;
      border-radius: 10px;
      box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
    }
  </style>
</head>
<body>
  <h1>Momentos de Decepción</h1>
  <p>Porque a veces es mejor recordarlo...</p>
  <div class="gallery">
    <img src="foto1.jpg" alt="Foto 1">
    <img src="foto2.jpg" alt="Foto 2">
    <img src="foto3.jpg" alt="Foto 3">
    
  </div>
</body>
</html>

**Comment by shinybrightstar** [comment]
Hi @izancabero , thanks for participating!

👋 and thank you for being a part of our community. To ensure a safe and supportive environment, we look to our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) which states all communication here on the Community needs to be in English. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be closing and locking any posts in languages other than English for now, including this one.
